boxoffice/models/discount_policy.py

```diff
@@ -37,9 +36,8 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
 
     discount_type = db.Column(db.Integer, default=DISCOUNT_TYPE.AUTOMATIC, nullable=False)
 
-    # Minimum and maximum number of items for which the discount policy applies
+    # Minimum number of a particular item that needs to be bought for this discount to apply
     item_quantity_min = db.Column(db.Integer, default=1, nullable=False)
-    item_quantity_max = db.Column(db.Integer, nullable=True)
```

Comment:
Will remove the `item_quantity_max` column from `DiscountPolicy` and the `quantity_available` and `quantity_total` columns from `DiscountCoupon` once I've made the data migration.